eva-server/src/test/java/uk/ac/ebi/eva/server/ws/ArchiveWSServerTest.java

```diff
@@ -290,6 +294,42 @@ public void testGetBrowsableStudiesBySpecies() throws URISyntaxException {
         }
     }
 
+    @Test
+    public void testGetBrowsableStudiesBySpeciesAndFromDate() throws URISyntaxException {
+        String url = "/v1/meta/studies/list?species=hsapiens_grch37&fromDate=2018-01-01";
+        ResponseEntity<QueryResponse<QueryResult<VariantStudySummary>>> response = restTemplate.exchange(
+                url, HttpMethod.GET, null,
+                new ParameterizedTypeReference<QueryResponse<QueryResult<VariantStudySummary>>>() {});
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+
+        QueryResponse<QueryResult<VariantStudySummary>> queryResponse = response.getBody();
+        assertEquals(1, queryResponse.getResponse().size());
+
+        List<VariantStudySummary> results = queryResponse.getResponse().get(0).getResult();
+        assertEquals(1, results.size());
+
+        for (VariantStudySummary variantStudySummary : results) {
+            assertTrue(variantStudySummary.getFilesCount() > 0);
+            assertEquals("Human Test study 1", variantStudySummary.getStudyName());
+            assertEquals("S1", variantStudySummary.getStudyId());
+        }
+    }
+
+    @Test
+    public void testGetBrowsableStudiesBySpeciesAndFutureFromDate() throws URISyntaxException {
+        String url = "/v1/meta/studies/list?species=hsapiens_grch37&fromDate=3000-01-01";
+        ResponseEntity<QueryResponse<QueryResult<VariantStudySummary>>> response = restTemplate.exchange(
+                url, HttpMethod.GET, null,
+                new ParameterizedTypeReference<QueryResponse<QueryResult<VariantStudySummary>>>() {});
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+
+        QueryResponse<QueryResult<VariantStudySummary>> queryResponse = response.getBody();
+        assertEquals(1, queryResponse.getResponse().size());
+
+        List<VariantStudySummary> results = queryResponse.getResponse().get(0).getResult();
+        assertEquals(0, results.size());
```

Comment:
this test failed in travis https://travis-ci.org/EBIvariation/eva-ws/builds/373408145#L4349